lib/ast-converter.js

```diff
@@ -115,22 +115,47 @@ TOKEN_TO_TEXT[SyntaxKind.CaretEqualsToken] = "^=";
 TOKEN_TO_TEXT[SyntaxKind.AtToken] = "@";
 TOKEN_TO_TEXT[SyntaxKind.InKeyword] = "in";
 
+/**
+ * Returns true if the given node is a valid ESTree class member
+ * @param  {object}  node AST node
```

Comment:
What I'd like to do is use `{ESTreeNode}` for anything that represents an ESTree AST node and `{TSNode}` for anything that represents a TypeScript AST node. In this case, it would be the latter.